Description: Upstream changes introduced in version 0.5-12
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 python-occ (0.5-12) natty; urgency=low
 .
   * recommends included
   * python2.7 only release
 .
 The person named in the Author field signed this changelog entry.
Author: Henning Meyer <hmeyer@gmx.eu>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- python-occ-0.5.orig/src/wrapper/environment.py
+++ python-occ-0.5/src/wrapper/environment.py
@@ -176,8 +176,8 @@ elif sys.platform=='linux2':
         DEFINE_MACROS.append(('_OCC64',None))
         SWIG_OPTS.append('-D_OCC64')
         ECA.append('-m64')
-    else:
-        ECA.append('-march=%s'%platform.machine())
+    #else:
+        #ECA.append('-march=%s'%platform.machine())
     
     lib_python = sysconfig.get_config_var('BLDLIBRARY').split(' ')[1]
     ELA = ['-Wl,--no-undefined','-lm','-lstdc++',lib_python]
--- python-occ-0.5.orig/AUTHORS
+++ python-occ-0.5/AUTHORS
@@ -7,3 +7,4 @@ Andy Haywood - andy@andyh.fsworld.co.uk
 Henrik Rudstrom - hrudstrom@googlemail.com
 Thomas Paviot - tpaviot@gmail.com
 
+packaged by Henning Meyer hmeyer@gmx.eu
--- python-occ-0.5.orig/src/customize_build.py
+++ python-occ-0.5/src/customize_build.py
@@ -37,6 +37,7 @@ def customize_compiler(compiler):
             compiler_so.remove('-g')
         #if '-O2' in compiler_so:
         #    compiler_so.remove('-O2')
+	compiler_so.append('-Os')
     elif sys.platform == 'win32':
         if not compiler.initialized:
             compiler.initialize()
